### ISSUE TITLE
Creating .json file 

### ISSUE THREAD

**Issue by Eliyanan** [joakimbeng/create-file#2]
can i create .json file using this module?

**Comment by joakimbeng** [comment]
Yes, you can create whatever you want. But you should always pass a string to this function (for the file contents) so use JSON.stringify first if you want to save an object as json.
